examples/user_interaction/client/src/index.js

```diff
@@ -45,6 +45,8 @@ const routing = (
 ReactDOM.render(routing, document.getElementById('root'));
 
 window.ocAgent = 'http://localhost:55678';
+// For the purpose of this example, just match all hosts.
```

Comment:
Nit: maybe say "For the purpose of this example, send trace header to all hosts"? (This will help people reading the example understand better what the field does).